v22.1/views.md

```diff
@@ -556,8 +556,50 @@ To remove the materialized view, use [`DROP MATERIALIZED VIEW`](drop-view.html):
 DROP VIEW
 ~~~
 
+### Add an index to a materialized view
+
+To speed up queries on materialized views, you can add an [index](schema-design-indexes.html) to the view.
+
+The following statement creates a materialized view of the [Movr](movr.html) rides table where the revenue is less than $20.00:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE MATERIALIZED VIEW low_rev_rides AS SELECT city, vehicle_id, revenue from rides WHERE revenue < 20.00;
+~~~
+
+The following statement creates an index on the city column:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE INDEX ON low_rev_rides (city) STORING (vehicle_id, revenue);
+~~~
+
+To see which vehicles earn revenue lower than $20.00 in Seattle, run the follow query:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SELECT vehicle_id, revenue FROM low_rev_rides@low_rev_rides_city_idx WHERE city = 'seattle';
+~~~
+
+~~~
+               vehicle_id              | revenue
+---------------------------------------+----------
+  55555555-5555-4400-8000-000000000005 |    3.00
+  55555555-5555-4400-8000-000000000005 |    7.00
+  55555555-5555-4400-8000-000000000005 |   18.00
+  55555555-5555-4400-8000-000000000005 |    7.00
+  55555555-5555-4400-8000-000000000005 |   15.00
+  66666666-6666-4800-8000-000000000006 |   11.00
+  66666666-6666-4800-8000-000000000006 |   11.00
+  66666666-6666-4800-8000-000000000006 |   16.00
+  55555555-5555-4400-8000-000000000005 |   13.00
+(9 rows)
```

Comment:
Rather than showing query results, would it make more sense to show the output of `EXPLAIN SELECT vehicle_id, revenue FROM low_rev_rides@low_rev_rides_city_idx WHERE city = 'seattle';` before and after the query is added so that the reader can see the different in query plan?